starter-code/src/components/ProductBox.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component} from 'react';
+
+export default class ProductBox extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      count: 1
+    };
+  }
+  
+  onChangeCount = (e) => {
+    this.setState({ count: e.target.value })
+  }
+
+  onAddToMenu = () => {
+    this.props.onAddToMenu({ ...this.props.food, quantity: this.state.count });
```

Comment:
Sabes que igualando la cantidad así, si no modificas la variable `count` no va a subir la cantidad del alimento, es decir, si tienes 1 y lo añades, si el contador previo tiene 1 también, estarás reemplazando ese 1 por el nuevo, pero no sumarías 1 a la cantidad. Para sumarle a la cantidad, en la función `onChangeCount` deberías darle a count el resultado de `value` + `count` de la siguiente manera `this.setState({ count: this.state.count + e.target.value })`